CONTRIBUTING.md

```diff
@@ -141,19 +141,22 @@ tracing) enabled.
 
 The setup for a local rustc works as follows:
 ```sh
+# Clone the rust-lang/rust repo.
 git clone https://github.com/rust-lang/rust/ rustc
 cd rustc
 cp config.toml.example config.toml
 # Now edit `config.toml` and set `debug-assertions = true`.
-# This step can take 30 minutes and more.
-./x.py build src/rustc
+
+# Build a stage 2 rustc.
+# This step can take 30 minutes or more.
+./x.py build compiler/rustc --stage 2
```

Comment:
I haven't test this particular command; I just used `./x.py build --stage 2`, but it's probably faster to use what this suggests.

Reply:
`compiler/rustc` is _slower_ than `build --stage 2`. https://rustc-dev-guide.rust-lang.org/building/bootstrapping.html#building-vs-running

Reply:
Revenge of the bootstrap stages! What should I change it to?

Reply:
```suggestion
./x.py build --stage 2 library/std
```